Xamarin.Forms.Platform.WinRT/WindowsDeviceInfo.cs

```diff
@@ -51,6 +51,9 @@ public override double ScalingFactor
 		{
 			get
 			{
+#if WINDOWS_UWP
+				return DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
```

Comment:
I think it would be better to use this for both platforms:
   `((int)_information.ResolutionScale) / 100d`

That ensures if the enum gets more options, it'll still work.

Reply:
I agree